SECURITY.md

```diff
@@ -387,7 +387,7 @@ If you are running in a runtime environment that also uses Netty (e.g., Hadoop,
 Below are known to work version combinations:
 
 grpc-netty version | netty-handler version | netty-tcnative-boringssl-static version
------------------- | --------------------- | ---------------------------------------
+------------------ |-----------------------| ---------------------------------------
```

Comment:
What happened here? Does some UI need it like this?

Reply:
I think IntelliJ tried to do something "clever" there. Fixed.

Reply:
Note that you extended the column width.